### DISCUSSION TITLE
Settings for HD-tDCS electrode parameters

### DISCUSSION THREAD

**Discussion by neogitter** [simnibs/simnibs#287] [6. Getting started]
Hi,

In our lab, we use Soterix 4x1 HD-tDCS with ring Ag/AgCL electrodes and gel. 

While trying to emulate the HD-tDCS stimulation in SimNIBS, I did not understand how to set the dimensions for the ring electrodes (Outer and inner diameter). I see only an option to set it as a disc. Would that make a difference in the simulations??

**Comment by TorgeW** [comment]
Hey,

Take a look at these examples :) They should show you how to define ring electrodes by using the inner and outer diameter:

Matlab:
https://github.com/simnibs/simnibs/blob/master/simnibs/examples/simulations/tDCS_ring.m

Python:
https://github.com/simnibs/simnibs/blob/master/simnibs/examples/simulations/tDCS_ring.py
